go/client/cmd_simplefs_copy.go

```diff
@@ -0,0 +1,143 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"errors"
+	"os"
+	"time"
+
+	"golang.org/x/net/context"
+
+	"encoding/hex"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+)
+
+// CmdSimpleFSCopy is the 'simplefs list' command.
+type CmdSimpleFSCopy struct {
+	libkb.Contextified
+	opid    keybase1.OpID
+	src     keybase1.Path
+	dest    keybase1.Path
+	recurse bool
+	argOpid bool // set when -o is used
+}
+
+// NewCmdSimpleFSCopy creates a new cli.Command.
+func NewCmdSimpleFSCopy(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:         "cp",
+		ArgumentHelp: "<source> [dest]",
+		Usage:        "copy directory elements",
+		Action: func(c *cli.Context) {
+			cl.ChooseCommand(&CmdDeviceList{Contextified: libkb.NewContextified(g)}, "cp", c)
+		},
+		Flags: []cli.Flag{
+			cli.BoolFlag{
+				Name:  "r, recursive",
+				Usage: "Recurse into subdirectories",
+			},
+			cli.StringFlag{
+				Name:  "o, opid",
```

Comment:
I don't think we want this second mode of operation in this or other CLI operations.  I think the CLI commands ought to be synchronous, and there's no concept of "resuming" them after the fact.  You can, if you want, call `keybase simplefs wait 458480ffgg` but I don't think we should handle a situation where you call the same command *twice* and the second invocation is to resume the first. That's way too foreign for UNIX interfaces.